include/xtypes/Module.hpp

```diff
@@ -357,6 +357,58 @@ class Module
         return result.second;
     }
 
+    AliasType& alias(
+            const std::string& name)
+    {
+        // Solve scope
+        PairModuleSymbol module = resolve_scope(name);
+        if (module.first == nullptr)
+        {
+            // This will fail
+            return static_cast<AliasType&>(const_cast<DynamicType&>(*module.first->aliases_.at(module.second)));
+        }
+
+        return static_cast<AliasType&>(const_cast<DynamicType&>(*module.first->aliases_.at(module.second)));
```

Comment:
Ok, we need to consider to make `operator *` `operator ->` and `.get()` non-const versions. We should avoid the `const_cast`

Reply:
Could you add this operations and the remove the `const_cast`?

Reply:
Done :+1: